mysql-test/main/func_str.test

```diff
@@ -2193,3 +2193,56 @@ SELECT GROUP_CONCAT( UpdateXML( '<a>new year</a>', '/a', '2019-01-01 00:00:00' )
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
+--echo #
+--echo # MDEV-25704 Function random_bytes
+--echo #
+
+--source include/have_sequence.inc
```

Comment:
test preconditions are usually at the beginning of the test so if they fail with a `skip test`, nothing much has been done.